maas/meta/backupninja.yml

```diff
@@ -0,0 +1,6 @@
+backup:
+  maas:
+    fs_includes: []
+    fs_excludes: []
+    actions:
+    - cmd: "rsync -aH  --exclude='/var/lib/maas/boot-resources' --relative /etc/maas /var/lib/maas /var/backups/maas_config"
```

Comment:
So use-case is to backup to multiple locations? Then you should add support into backupninja for this use-case, this behavior may not be desired for all deployments as MaaS data can be probably pretty heavy (all the images, etc.). Purpose of /var/backups is to have place where to collect some data before sending them to backup location (eg. data that are continuously changing, et mysql database that you need to dump first). But it's not meant to be persistent or somehow reliable backup location.